libnopegl/src/backends/vk/rendertarget_vk.c

```diff
@@ -328,6 +328,7 @@ static VkResult rendertarget_vk_init(struct rendertarget *s)
             return res;
 
         s_priv->attachments[s_priv->nb_attachments] = view;
+        s_priv->attachments_refs[s_priv->nb_attachments] = NGLI_RC_REF(texture);
```

Comment:
I'm a bit confused why this is not done above for the color attachment

Reply:
Me too :) The chunk has simply been forgotten.